# HG changeset patch
# User Betterbird <betterbird@betterbird.eu> and DeepSeek
# Parent  92ba6d7df2ca46581c35a59877320cf9a6e9fd84
Branding: codesign --force --deep --sign - Betterbird.app

diff --git a/toolkit/mozapps/installer/packager.mk b/toolkit/mozapps/installer/packager.mk
--- a/toolkit/mozapps/installer/packager.mk
+++ b/toolkit/mozapps/installer/packager.mk
@@ -129,8 +129,12 @@ ifeq (Darwin_cocoa, $(OS_ARCH)_$(MOZ_WID
 endif # Darwin_cocoa
 endif # MOZ_ARTIFACT_BUILDS
 
 make-package-internal: prepare-package make-sourcestamp-file
+ifeq (Darwin,$(OS_ARCH))
+	@echo 'Ad-hoc signing $(MOZ_MACBUNDLE_NAME).'
+	codesign --force --deep --sign - '$(DIST)/$(MOZ_PKG_DIR)/$(MOZ_MACBUNDLE_NAME)'
+endif
 	@echo 'Compressing...'
 	$(call MAKE_PACKAGE,$(DIST))
 	echo $(PACKAGE) > $(ABS_DIST)/package_name.txt
 
